packages/dev/gui/src/2D/advancedDynamicTexture.ts

```diff
@@ -364,6 +364,11 @@ export class AdvancedDynamicTexture extends DynamicTexture {
         this._clipboardData = value;
     }
 
+    /**
+     * If this is set, even when a control is pointer blocker, some events can still be passed through to the scene.
+     */
+    public skipBlockEvents = 0;

```

Comment:
maybe this should be types as "one of PointerEventTypes"? I noticed it is a class wiath static elements, so it might not be too trivial, but it will be simpler to understand where the number(s) that I can use should come from.

Reply:
@RaananW I couldn't find a better way to add it, so I added in the docstring